### PR TITLE
feat: image sagemaker client updates

### PR DESCRIPTION
**Pull request by zanussbaum**

### CODE REVIEW

File: examples/sagemaker/run-nomic-embed-vision.ipynb [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b29175ef",
+   "metadata": {},
+   "source": [
+    "## Batch Transform\n",
+    "\n",
+    "Nomic Embed on Sagemaker also supports [batch transform jobs](https://docs.aws.amazon.com/sagemaker/latest/dg/batch-transform.html) over S3 buckets.\n",
+    "To run batch transform, make sure your bucket PNG or JPEGs.\n",
```
Comment (rguo123) [inline examples/sagemaker/run-nomic-embed-vision.ipynb:100]: nit 
```suggestion
    "To run batch transform, make sure your bucket contains PNG or JPEGs.\n",
```

File: nomic/aws/sagemaker.py [hunk 2]
```diff
@@ -180,3 +185,139 @@ def embed_texts(
...
+
+    client = boto3.client("sagemaker-runtime", region_name=region_name)
+    response = client.invoke_endpoint(
+        EndpointName=sagemaker_endpoint, Body=preprocessed_image[0], ContentType="image/jpeg"
+    )
+
+    return parse_sagemaker_response(response)
+
+
+def embed_image(
```
Comment (rguo123) [inline nomic/aws/sagemaker.py:229]: nit: `embed_images` instead?

**Review by rguo123 — APPROVED** [review]
LGTM!

File: nomic/aws/sagemaker.py (hunk 2 above)
Comment (zanussbaum) [reply nomic/aws/sagemaker.py:229]: yeah it's a little odd our python client has `embed.text` as well as `embed.images` while in sagemaker we have `embed_texts` and `embed_image`. I'll fix to mirror what our api has, thanks for pointing this out!

Comment (AndriyMulyar) [reply nomic/aws/sagemaker.py:229]: we should change our python client signature to embed.image

Comment (zanussbaum) [reply nomic/aws/sagemaker.py]: why?

Comment (zanussbaum) [reply nomic/aws/sagemaker.py:229]: updated

**Pull request merged**